lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java

```diff
@@ -42,18 +43,46 @@
 	private MavenPluginUtils() {
 		// Utility class, not meant to be instantiated
 	}
+	
+	public static String convertToBold(String message, boolean supportsMarkdown) {
```

Comment:
It looks like this method does more or less nothing when supportsMarkdown=false.
I suggest we just don't call it in such case.
Also, such methods would better fit in a new MarkdownUtils class or something like that.